editor/import/scene_import_settings.cpp

```diff
@@ -530,6 +537,10 @@ void SceneImportSettings::_load_default_subresource_settings(HashMap<StringName,
 	}
 }
 
+void SceneImportSettings::request_generate_collider(ResourceImporterScene::InternalImportCategory &category) {
```

Comment:
Why does this pass a category as argument if it isn't used?

Reply:
Thanks for the catch, just cause when making a change Clay requested I forgot to remove the argument. Just fixed it